README.md

```diff
@@ -154,7 +154,7 @@ platforms are:
 - DOS (Allegro)
 - FreeBSD (SDL)
 - Linux (SDL or Allegro)
-- MacOS X (universal) (Cocoa video and sound drivers)
+- macOS (universal) (Cocoa video and sound drivers)
```

Comment:
You can probably get rid of `(universal)` as well, this dates back from the PPC/x86 transition.

Reply:
OK. I'll remove it in next commit. Thanks for noticing :)

Reply:
Looking into the Download Page you can notice the following description: `Mac OS X 10.3.9-10.5 (universal build)`. It contains word "universal", so I'm not sure I should remove it in next commit. It is also possible that this description is obsolete.

Reply:
that should still be correct, at least in the respect that it packs 32bit/64bit. So keep it (for now)

Reply:
> that should still be correct, at least in the respect that it packs 32bit/64bit

That is not what 'universal' means, though, in this context. It's universal if it contains code for both PPC and x86.

Reply:
I made a new commit in which I left this as it is (for now?).